### ISSUE TITLE
Accessibility Review for Claimant Search

### ISSUE THREAD

**Issue by candilemoine** [department-of-veterans-affairs/va.gov-team#109637]
#### Overview
We will complete a [full accessibility review](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review) of the Claimant Search, prior to going to staging review. 

As an accredited representative, I would like search for a specific claimant, so I can find any pending or existing POA requests.

#### Resources

- Local URL:`http://staging.va.gov/representative/representative-upload-form/686c`

### Acceptance Criteria/Conditions of Satisfaction

- [ ] Verify test user account to use 
- [ ] Fill out the audit document and log all issues in there
- [ ] Audit using NVDA/Chrome
- [ ] Audit using Voiceover/Safari on Desktop
- [ ] Audit using Android Talkback on chrome
- [ ] Audit using either iPad or iPhone with Voiceover/Safari mobile

----

Platform guidance: https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review

### Who completed the use of color and color contrast test? 
[Tester] tested with the Chrome browser for both desktop and mobile (Android).
[Tester] tested with the Safari browser for both desktop (Macbook Air) and mobile (iPhone 13 Pro).

### Use of color and color contrast checks
Tools used: Wave and Axe extensions for Chrome, Colour Contrast Analyzer for Mac.

- [ ] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [ ] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [ ] Non-text elements have a 3:1 contrast ratio to their background and to neighboring elements (or better)
- [ ] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [ ] Any charts, maps, infographics, and tables convey all information without only relying on color
- [ ] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### How did color testing go?
Chrome:

Safari:

### Who completed the axe scans?
[Tester]

Axe Checks:

- [ ] Each page has been scanned using axe (results shared below)
- [ ] axe is integrated into your end-to-end testing

### Who completed the content zoom and reflow test?
[Tester] tested with the Chrome browser for both desktop and mobile (Android).
[Tester] tested with the Safari browser for both desktop (Macbook Air) and mobile (iPhone 13 Pro).

### Content zoom and reflow checks (desktop AND mobile)
**Desktop:**

- [ ] All page elements are readable and usable at 200% zoom
- [ ] All page elements are readable and usable at 300% zoom
- [ ] All page elements are readable and usable at 400% zoom

Successfully tested on desktop (Chrome and Safari) and all page elements are readable and usable at 200, 300, and 400% zoom.

**Mobile:**

- [ ] All page elements are readable and usable at 200% zoom
- [ ] All page elements are readable and usable at 300% zoom
- [ ] All page elements are readable and usable at 400% zoom Chrome on Android and Safari on iPhone both have a limit of 300%.

For both mobile browsers, many page elements are not readable and usable at 200% and 300% zoom. This includes:

- Issues with text not wrapping / getting cut off
- Issues with resizing
- Issues with content taking up the entire screen (Safari on iPhone at 300%)

More details, including screenshots, in the comments below.

### Who completed the keyboard navigation test?
[Tester] tested with the Chrome browser.
[Tester] tested with the Safari browser.

### Keyboard navigation checks

- [ ] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [ ] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [ ] All elements under focus have a visible focus indicator
- [ ] The order of [Tab] stops made sense and was appropriate for completing tasks

### Do you have any other results to share?

**Comment by jquispe-oddball** [comment]
@candilemoine I found a few issues so far. I am blocked by not being able to search for a claimant. I can test the default page and "user not found" view.

This issue was actually fixed with this ticket from last week - https://github.com/department-of-veterans-affairs/va.gov-team/issues/109937
![Image](https://github.com/user-attachments/assets/5977bcaf-3c7c-4d2e-84e9-56d98cd05b7e)

The heading needs to be updated to an h2 with an h3 css class 
<img width="1785" alt="Image" src="https://github.com/user-attachments/assets/1fb982b4-769b-43f2-9b7f-a9c43c79bc1e" />

**Comment by jquispe-oddball** [comment]
I was able to test with a user and did not see any accessibility errors.

<img width="1784" alt="Image" src="https://github.com/user-attachments/assets/899a2dce-6f3f-4f77-9550-7928419fe2dd" />
